packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -14,7 +14,7 @@
 import * as vscode from "vscode";
 import * as fs from "fs";
```

Comment:
This one still shows as unused for me @zFernand0, but I am approving to get this in. We can clean up when we add eslint perhaps.

Reply:
the `IConfigLayer` really ? 🤔 
https://github.com/zowe/vscode-extension-for-zowe/blob/450db8fa9a362ee35618a9682ccce7e3741b514d/packages/zowe-explorer/src/utils/ProfilesUtils.ts#L256

Reply:
No, the fs import